homeassistant/package_constraints.txt

```diff
@@ -189,3 +189,6 @@ charset-normalizer==3.2.0
 # lxml 5.0.0 currently does not build on alpine 3.18
 # https://bugs.launchpad.net/lxml/+bug/2047718
 lxml==4.9.4
+
+# dacite: Ensure we have a version that is able to handle type unions for Roborock.
```

Comment:
Maybe we should add information here that NAM, Brother and GIOS integrations also use `dacite`?